src/realm/parser/driver.cpp

```diff
@@ -16,6 +16,71 @@ static bool trace_scanning = false;
 
 namespace {
 
+const char* data_type_to_str(DataType type)
```

Comment:
We should not add a competing function, but rather modify that in table.cpp

Reply:
thanks for pointing out the duplication, I guess I forgot we already had this!